components/application-mgt/org.wso2.carbon.device.application.mgt.core/src/main/java/org/wso2/carbon/device/application/mgt/core/dao/impl/Comment/CommentDAOImpl.java

```diff
@@ -0,0 +1,1018 @@
+
+        if (log.isDebugEnabled()) {
+            log.debug("Getting comment with the application release(" + uuid + ") from the database");
+        }
+        Connection conn;
+        PreparedStatement stmt = null;
+        ResultSet rs;
+        String sql = "";
+        List<Comment> comments = new ArrayList<>();
+        try {
+            conn = this.getDBConnection();
+            sql += "SELECT COMMENT_TEXT FROM AP_APP_COMMENT WHERE (SELECT ID FROM AP_APP_RELEASE where UUID=?)AND "
+                    + "(SELECT AP_APP_ID FROM AP_APP_RELEASE where UUID=?);";
+            stmt = conn.prepareStatement(sql);
+            stmt.setString(1, uuid);
+            stmt.setString(2, uuid);
+            rs = stmt.executeQuery();
+            while (rs.next()) {
+                Comment comment = new Comment();
+                comment.setId(rs.getInt("ID"));
+                comment.setTenantId(rs.getInt("TENANT_ID"));
+                comment.setComment(rs.getString("COMMENT_TEXT"));
+                comment.setCreatedAt(rs.getTimestamp("CREATED_AT"));
+                comment.setCreatedBy(rs.getString("CREATED_BY"));
+                comment.setModifiedAt(rs.getTimestamp("MODEFIED_AT"));
+                comment.setModifiedBy(rs.getString("MODEFIED_AT"));
+                comment.setParent(rs.getInt("PARENT_ID"));
+                comments.add(comment);
+            }
+        } catch (DBConnectionException e) {
+            log.error("DB Connection Exception occurred while retrieving comments", e);
+        } catch (SQLException e) {
+            throw new CommentManagementException("SQL Error occurred while retrieving comments", e);
+        } finally {
+            Util.cleanupResources(stmt, null);
+        }
+        return comments;
+    }
+
+    @Override
+    public List<Comment> getAllComments(String uuid, PaginationRequest request)
+            throws CommentManagementException, SQLException, DBConnectionException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Getting comment of the application release (" + uuid + ") from the database");
+        }
+        Connection conn;
+        PreparedStatement stmt = null;
+        ResultSet rs = null;
+        List<Comment> comments = new ArrayList<>();
+        String sql = "";
+        try {
+            conn = this.getDBConnection();
+            sql += "SELECT COMMENT_TEXT FROM AP_APP_COMMENT, AP_APP_RELEASE WHERE AP_APP_COMMENT.AP_APP_RELEASE_ID=AP_APP_RELEASE.ID "
+                    + "AND AP_APP_RELEASE.UUID =? LIMIT ? OFFSET ?;";
+            stmt = conn.prepareStatement(sql);
+            stmt.setString(1, uuid);
+            stmt.setInt(2, request.getLimit());
+            stmt.setInt(3, request.getOffSet());
+            rs = stmt.executeQuery();
+            while (rs.next()) {
+                Comment comment = new Comment();
+                comment.setId(rs.getInt("ID"));
+                comment.setTenantId(rs.getInt("TENANT_ID"));
```

Comment:
@NishadiPrasangini Since you have used these string values multiple times within the same file, it's better to define these as a constant.